<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myOrder</class>
 <widget class="QWidget" name="myOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 81, 100);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>881</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 51, 62);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>订单号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>起点</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>终点</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>航班日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下单日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>购买数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总金额</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>订单状态</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-right-radius:15px;
border-bottom-right-radius:15px;</string>
   </property>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 年 |</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 月 |</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px;</string>
   </property>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton:!hover
{
background-color: rgb(255, 255, 255);
border-top-right-radius:15px;
border-bottom-right-radius:15px;
}
#pushButton:hover
{
background-color: rgb(85, 187, 138);
border-top-right-radius:15px;
border-bottom-right-radius:15px;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2:!hover
{
background-color: rgb(255, 255, 255);
border-radius:15px;

}
#pushButton_2:hover
{
background-color: rgb(85, 187, 138);
border-radius:15px;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>查询全部订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3:!hover
{
background-color: rgb(255, 255, 255);
border-top-left-radius:15px;
border-bottom-left-radius:15px;
}
#pushButton_3:hover
{
background-color: rgb(85, 187, 138);
border-top-left-radius:15px;
border-bottom-left-radius:15px;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>未出行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_4:!hover
{
background-color: rgb(255, 255, 255);
border-radius:15px;
}
#pushButton_4:hover
{
background-color: rgb(85, 187, 138);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>已出行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_5:!hover
{
background-color: rgb(255, 255, 255);
border-radius:15px;
}
#pushButton_5:hover
{
background-color: rgb(85, 187, 138);
color: rgb(255, 255, 255);
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>已失败</string>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
